identity/user.go

```diff
@@ -0,0 +1,56 @@
+package identity
+
+import (
+	"github.com/pkg/errors"
+	"perun.network/go-perun/wallet"
+)
+
+// WalletBackend defines the methods that need to be implemented by a blockchain platform specific wallet backend.
+type WalletBackend interface {
+	NewWallet(string, string) (wallet.Wallet, error)
+	NewAccount(wallet.Wallet, wallet.Address) (wallet.Account, error)
+}
+type User struct {
+	Peer
+
+	// Account for funding the channel and the onchain transactions.
+	OnChainAcc wallet.Account
+	// OnChainKeystorePath is the wallet that stores the keys for onchain account.
+	OnChainWallet wallet.Wallet
+
+	// Account corresponding to perunID.
+	// Used for authenticating user identity in offchain network.
```

Comment:
Im not sure if this comment makes sense.

Reply:
Updated.